src/mixins/dom.js

```diff
@@ -3,17 +3,28 @@
 
 import Backbone from 'backbone';
 
+function getEl(el, context) {
+  return el instanceof Backbone.$ ? el : Backbone.$(el, context);
```

Comment:
I dunno maybe we'll find it is still useful..  I _would_ like to remove the `parentEl` stuff from Region however.. and the only other contextual `getEl` call is in `CompositeView`, but I could also see utilizing the `DomApi` to replace the `view.$()` implementation..